Geometry/HGCalCommonData/plugins/dd4hep/DDHGCalSiliconModule.cc

```diff
@@ -0,0 +1,426 @@
+///////////////////////////////////////////////////////////////////////////////
+// File: DDHGCalSiliconModule.cc
+// Description: Geometry factory class for HGCal (EE and HESil) using
+//              information from the file for dd4hep
+///////////////////////////////////////////////////////////////////////////////
+#include <cmath>
+#include <memory>
+#include <sstream>
+#include <string>
+#include <unordered_set>
+#include <vector>
+
+#include "Geometry/HGCalCommonData/interface/HGCalGeomTools.h"
+#include "Geometry/HGCalCommonData/interface/HGCalParameters.h"
+#include "Geometry/HGCalCommonData/interface/HGCalProperty.h"
+#include "Geometry/HGCalCommonData/interface/HGCalTypes.h"
+#include "Geometry/HGCalCommonData/interface/HGCalWaferIndex.h"
+#include "Geometry/HGCalCommonData/interface/HGCalWaferType.h"
+#include "DD4hep/DetFactoryHelper.h"
+#include "DataFormats/Math/interface/CMSUnits.h"
```

Comment:
Please change the include to `angle_units.h`. DD4hep algorithms cannot be tied to CMS units anymore.

Reply:
Done